test/e2e/integration/005-splitting-merging-paragraph-blocks.js

```diff
@@ -0,0 +1,24 @@
+describe( 'Splitting and merging paragraph blocks', () => {
```

Comment:
It seems like it could be part of the existing test suite: https://github.com/twsp/gutenberg/blob/0f4035b0b266390f9e8facc05037175180dceb67/test/e2e/integration/002-adding-blocks.js

Reply:
I'm going to work a little more on this and then submit another pull request.  Should I keep the testing of splitting/merging in a separate file, or add it onto 002-adding-blocks.js?

Reply:
I think it fits the existing test file, so you can put it there 👍